hdfs-commons/src/main/java/org/apache/mesos/hdfs/util/FailureUtils.java

```diff
@@ -0,0 +1,19 @@
+package org.apache.mesos.hdfs.util;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+/**
+ * Failure utilities.
+ */
+public class FailureUtils {
+  private static final Log log = LogFactory.getLog(FailureUtils.class);
+
```

Comment:
Instead of suppressing the warnings here, would it make sense to update the gradle settings to be more lenient. I'd rather avoid suppressing warnings if possible. Also, I'm curious about why we need to suppress the warnings here. I see we're calling System.exit in our executors now, https://github.com/mesosphere/hdfs/blob/master/hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java#L190.